packages/plugin-packer/src/cli.ts

```diff
@@ -1,27 +1,20 @@
-"use strict";
-
-const path = require("path");
-const fs = require("fs");
-const os = require("os");
-const chokidar = require("chokidar");
-const denodeify = require("denodeify");
-
-const writeFile = denodeify(fs.writeFile);
-const mkdirp = require("mkdirp");
-const debug = require("debug")("cli");
-const validate = require("@kintone/plugin-manifest-validator");
-
-const packer = require("./");
-const console = require("./console");
-const generateErrorMessages = require("./gen-error-msg");
-const createContentsZip = require("./create-contents-zip");
-
-/**
- * @param {string} pluginDir path to plugin directory.
- * @param {Object=} options {ppk: string, out: string}.
- * @return {!Promise<string>} The resolved value is a path to the output plugin zip file.
- */
-function cli(pluginDir, options_) {
+import path from "path";
+import fs from "fs";
+import { promisify } from "util";
+import os from "os";
+import * as chokidar from "chokidar";
+import mkdirp from "mkdirp";
+import _debug from "debug";
+import validate from "@kintone/plugin-manifest-validator";
+import packer from ".";
+import console from "./console";
+import { generateErrorMessages } from "./gen-error-msg";
+import { createContentsZip } from "./create-contents-zip";
+
+const debug = _debug("cli");
+const writeFile = promisify(fs.writeFile);
+
+export = function cli(pluginDir: string, options_?: any) {
```

Comment:
`packer` takes a Buffer as an argument. So you have to fix the type of `createContentsZip` return value.

```suggestion
export = function cli(
  pluginDir: string,
  options_?: Partial<{
    ppk: string;
    out: string;
    watch: boolean;
    packerMock_: typeof packer;
  }>
) {
```

Reply:
5d677b2 thanks!